artifact/gem5art/artifact/_artifactdb.py

```diff
@@ -179,17 +179,17 @@ def searchByLikeNameType(self, name: str, typ: str, limit: int) -> Iterable[Dict
         for d in data:
             yield d
 
-_db = None
+db = None
```

Comment:
I think we want to keep this as `_db` because it's private. I hope I'm not contradicting my previous statement here.

Reply:
updated!